PowerUp/app/src/main/java/powerup/systers/com/data/entities/Accessory.java

```diff
@@ -1,10 +1,12 @@
 package powerup.systers.com.data.entities;
 
 import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
 
 @Entity(tableName = "Accessories")
 public class Accessory {
 
+    @PrimaryKey
     private Integer accessoryId;
```

Comment:
use "int" instead of "Integer" where appropriate unless you specifically need to use it ??, its used at multiple places, please take care of this.